qa/tasks/persistent_write_log_cache_thrash.py

```diff
@@ -0,0 +1,99 @@
+"""
+persistent write log cache thrash task
+"""
+import contextlib
+import logging
+import random
+
+from teuthology import misc as teuthology
+from teuthology import contextutil
+from teuthology.orchestra import run
+
+DEFAULT_KILL_DAEMON_TIME = 20
+io_pattern = ("seq", "rand")
+io_size = ('4', '16', '128', '1024')
+
+log = logging.getLogger(__name__)
+
+@contextlib.contextmanager
+def thrashes_rbd_bench_on_persistent_cache(ctx, config):
+    """
+    thrashes rbd bench on persistent write log cache.
+    It can test recovery feature of persistent write log cache.
+    """
+    log.info("thrashes rbd bench on persistent write log cache")
+
+    client, client_config = list(config.items())[0]
+    (remote,) = ctx.cluster.only(client).remotes.keys()
+    client_config = client_config if client_config is not None else dict()
+    image_name = client_config.get('image_name', 'testimage')
+    cache_path = client_config.get('rbd_persistent_cache_path', '/home/ubuntu/cephtest/rbd-pwl-cache')
+    kill_daemon_time = client_config.get('kill_daemon_time', DEFAULT_KILL_DAEMON_TIME)
+    ls_command = '"$(ls {} )"'.format(cache_path)
+    rbd_status_command = '"$(rbd status testimage | grep "Persistent cache state")"'
```

Comment:
This seems to hard-code the image name?

Reply:
Also, instead of piping to `grep`, I would just run `rbd status` and look for `Persistent cache state` in the output using Python.  This way if `rbd status` fails for some reason, it would be easier to notice.